sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1003,6 +1003,13 @@ def _add_argparse_args(cls, parser):
             'environment. Choose the docker build engine of local docker '
             'environment or google cloud build by passing the option '
             'local_docker or cloud_build.'))
+    parser.add_argument(
+        '--prebuild_sdk_container_base_image',
+        default=None,
+        help=(
+            'The base image to use when pre-building the sdk container image '
+            'with dependencies, if not specified, the latest public '
```

Comment:
For a dev sdk version, can we use the last released version?

Reply:
It may be incompatible when breaking changes happen. We could require that for dev sdks users must provide an image (could be 'nightly' if/when we build those, or built from Dockerfile).

Reply:
I agree that we should use a nightly build and we can change to that if we build those. Some context is that we may need to use this feature in some runner v2 rolling upgrade test and building from dockerfile for the test seems not applicable.